generate_protos.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+OUTPUT=./chainlibpy/generated
+COSMOS=./cosmos-sdk
+TENDERMINT=./tendermint
+TMP=$(whereis grpc_python_plugin)
+PLUGIN="$(cut -d' ' -f2 <<<"$TMP")"
+
+rm -rf $OUTPUT
+mkdir $OUTPUT
+git clone --branch v0.44.0 git@github.com:cosmos/cosmos-sdk.git $COSMOS
+git clone --branch v0.34.10 https://github.com/tendermint/tendermint.git $TENDERMINT
+# cosmos
+python -m grpc.tools.protoc --proto_path=$COSMOS/proto --proto_path=$COSMOS/third_party/proto --python_out=$OUTPUT $(find $COSMOS/proto/cosmos -iname "*.proto") --grpc_python_out=$OUTPUT  --plugin=protoc-gen-grpc_python=$PLUGIN
+# cosmos third-party
+python -m grpc.tools.protoc --proto_path=$COSMOS/proto --proto_path=$COSMOS/third_party/proto --python_out=$OUTPUT $(find $COSMOS/third_party/proto -iname "*.proto") --grpc_python_out=$OUTPUT  --plugin=protoc-gen-grpc_python=$PLUGIN
```

Comment:
just need to generate what we need instead of "*.proto", see:
https://github.com/crypto-org-chain/chain-jslib/blob/dev/lib/src/cosmos/v1beta1/scripts/get-proto.sh
https://github.com/crypto-org-chain/chain-jslib/blob/dev/lib/src/cosmos/v1beta1/scripts/predefine-proto.sh

Reply:
The latest commit will let developer add path to `.proto` files needed in `generate_protos.sh`. So we only generate gRPC code that we need instead of everything.